### DISCUSSION TITLE
Why does :timing return False?

### DISCUSSION THREAD

**Discussion by dclong** [evcxr/evcxr#228] [Q&A]
I used the command `:timing` to measure the execution time of a long-running cell, but it return "Timing: False" (not error) instead of outputting the execution time. Why is this? How can I force `:timing` to always output execution time?

**Comment by davidlattimore** [comment]
The `:timing` command doesn't take any argument, it just toggles timing on and off. When timing is on, all subsequent code will get timed. I should probably make it report an error if you try to give an argument to the `:timing` command.
